composer.json

```diff
@@ -23,19 +23,22 @@
     "require": {
         "php": "^7.1",
         "doctrine/phpcr-bundle": "^1.3 || ^2.0",
-        "doctrine/phpcr-odm": "^1.4 || ^2.0",
+        "doctrine/phpcr-odm": "^1.4.2 || ^2.0",
         "sonata-project/admin-bundle": "^3.30.1",
         "sonata-project/block-bundle": "^3.11",
         "symfony-cmf/resource-rest-bundle": "^1.0.1",
         "symfony-cmf/tree-browser-bundle": "^2.0",
-        "symfony/config": "^2.8 || ^3.2",
-        "symfony/dependency-injection": "^2.8 || ^3.2",
-        "symfony/form": "^2.8.8 || ^3.2",
-        "symfony/framework-bundle": "^2.8.8 || ^3.2",
-        "symfony/http-foundation": "^2.8 || ^3.2",
-        "symfony/http-kernel": "^2.8 || ^3.2",
-        "symfony/options-resolver": "^2.8 || ^3.2",
-        "symfony/property-access": "^2.8 || ^3.2"
+        "symfony/config": "^2.8 || ^3.2 || ^4.0",
+        "symfony/dependency-injection": "^2.8 || ^3.2 || ^4.0",
+        "symfony/finder": "^2.8 || ^3.2 || ^4.0",
+        "symfony/form": "^2.8 || ^3.2 || ^4.0",
+        "symfony/framework-bundle": "^2.8 || ^3.2 || ^4.0",
+        "symfony/http-foundation": "^2.8 || ^3.2 || ^4.0",
+        "symfony/http-kernel": "^2.8 || ^3.2 || ^4.0",
+        "symfony/options-resolver": "^2.8 || ^3.2 || ^4.0",
+        "symfony/property-access": "^2.8 || ^3.2 || ^4.0",
+        "sebastian/environment": "^1.3.4",
+        "sebastian/exporter": "^2.0"
```

Comment:
This is not sorted hence the FlintCI issue. Also, why did you touch this? Also, should really this be in the `require` section, as opposed to the `require-dev` section?